__tests__/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,11 @@
 const { booleanToWord } = require('../src');
 
 describe('booleanToWord', () => {
-  // how do we create specs again???
-});
+  it('should return Yes if true', () => {
+  expect(booleanToWord(true)).toEqual('Yes')
```

Comment:
Ensure consistency in indentation. Either 2 spaces or 4 spaces when inside a block of code. I recommend 2 spaces as it's more widely adopted in JS.